lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -253,6 +258,11 @@ class ReportRenderer {
     reportContainer.appendChild(reportSection);
     reportSection.appendChild(this._renderReportFooter(report));
 
+    if (fullPageScreenshot) {
```

Comment:
can this go up with the fullPageScreenshot code up on line 196 to keep it all together?